tfjs-layers/src/backend/random_seed.ts

```diff
@@ -16,11 +16,14 @@
 
 export class RandomSeed {
   static className = 'RandomSeed';
-  currentSeed: number;
-  constructor(readonly seed: number) { 
-    this.currentSeed = seed; 
+  seed: number | undefined;
+  constructor(seed: number | undefined) { 
+    this.seed = seed; 
   }
-  next() { 
-    ++this.currentSeed; 
+  next(): number | undefined { 
+    if (this.seed === undefined) {
+      return undefined;
+    }
+    return this.seed++; 
```

Comment:
Note to other reviewer: When no seed is specified, the keras implementation does not choose a random seed. It just generates random values. Saving it yields something like this, with the seed set to `Null`:
```
{'name': 'random_width_3', 'trainable': True, 'dtype': 'float32', 'factor': (-0.2, 0.3), 'interpolation': 'bilinear', 'seed': None}
```
This is truly random, and there is no reproducibility here.

On the other hand, when the user sets the seed, it gets saved to the config.
```
{'name': 'random_width_2', 'trainable': True, 'dtype': 'float32', 'factor': (-0.2, 0.3), 'interpolation': 'bilinear', 'seed': 1}
```